Source/JCC/config.cpp

```diff
@@ -442,6 +1106,30 @@ class CfgPatches
 		
 	
 
+		
```

Comment:
Too many new-lines. Remove all these. There should be only **one** new-line at the end of your code.